content-src/lib/getVideoPreview.js

```diff
@@ -1,32 +0,0 @@
-const getYouTubeID = require("get-youtube-id");
```

Comment:
FWIW, I ~~really~~ like the [**depcheck**](https://www.npmjs.com/package/depcheck) module for tracking unused modules. Although it can be somewhat verbose when trying to ignore lots of false positives (or things that aren't strict code modules but more ESLint/Babel configs or CLI tools):

```sh
$ depcheck --ignores=babel*,karma*,npm-run-all,sass-lint,rimraf,jpm,husky,svgo,eslint*,conventional-changelog-cli,cpx,node-sass,mocha,sinon,json-loader

Unused dependencies
* get-youtube-id
* redux-logger

Unused devDependencies
* eventemitter2
* istanbul-instrumenter-loader
* raw-loader
* requirejs
* style-loader

Missing dependencies
* eslint-plugin-json
* eslint-plugin-mozilla
* eslint-plugin-promise
* eslint-plugin-react
* chrome
* sdk
* addon
* common
* @loader/options
* selectors
* components
* test
* lib
* inject!addon
* shims
```